### PR TITLE
Form footer

### PR DESCRIPTION
**Pull request by Gunjuzone**
In this pull request, form and footer are added to my portfolio.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,7 +29,223 @@ <h3>I am a Software Developer</h3>
...
+      </div>        
+  </section> 
+
+  <section class="form-footer">
+    <div class="form-description">
+      <h2>I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.</h2>
+    </div>
+
+    <form action="https://formspree.io/f/mleanrp" method="POST">
```
Comment (Davidosky007) [inline index.html]: I suggest you check the form id properly i get this when i submit: 
Screenshot (75)

File: index.html [hunk 2]
```diff
@@ -30,7 +29,223 @@ <h3>I am a Software Developer</h3>
...
+
+     <footer  class="footer-links">
+      <ul>
+        <li class="footer-link"><a href="#"> <img src="Assets/githubb.png" alt=""></a></li>
+        <li class="footer-link"><a href="#"> <img src="Assets/Linkedinn.png" alt=""></a></li>
+        <li class="footer-link"><a href="#"> <img src="Assets/Vectorr.png" alt=""></a></li>
+        <li class="footer-link"><a href="#"> <img src="Assets/twitterr.png" alt=""></a></li>
+        <li class="footer-link"><a href="#"> <img src="Assets/mediumm.png" alt=""></a></li>
+      </ul>
+    </footer>
```
Comment (Davidosky007) [inline index.html]: This section is different from the original design template please fix it 🙏 
* Yours:
Screenshot (76)

* Figma Design:
Screenshot (77)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi, @Gunjuzone, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
* I suggest you make your pull request have a descriptive title telling what changes were made inside.
* And a short summary describing introduced changes in more detail.
* I left you comments below please make the necessary changes 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @Gunjuzone , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

## Something to include ☑️ 
- Your responsive is good, but when I see the desktop size it looks like this, I kindly invite you to change this.
image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**